### DISCUSSION TITLE
Tailwind not working on fresh vue-cli-service & Vite Vue 3 project

### DISCUSSION THREAD

**Discussion by NTKrieger** [tailwindlabs/tailwindcss#5134] [Help]
I followed all the installation instructions carefully, but can't get any of the styles to work.   postcss.config.js, tailwind.config.js, and index.css are all exactly as shown here:

https://tailwindcss.com/docs/guides/vue-3-vite

```js
// ./package.json
{
  "name": "metamind",
  "version": "0.1.0",
  "private": true,
  "scripts": {
    "serve": "vue-cli-service serve",
    "build": "vue-cli-service build",
    "lint": "vue-cli-service lint",
    "dev": "vite",
    "vite": "node ./bin/vite"
  },
  "dependencies": {
    "autoprefixer": "^10.3.1",
    "core-js": "^3.6.5",
    "postcss": "^8.3.6",
    "tailwindcss": "^2.2.7",
    "vue": "^3.0.0",
    "vue-router": "^4.0.0-0"
  },
  "devDependencies": {
    "@vue/cli-plugin-babel": "~4.5.0",
    "@vue/cli-plugin-eslint": "~4.5.0",
    "@vue/cli-plugin-router": "~4.5.0",
    "@vue/cli-service": "~4.5.0",
    "@vue/compiler-sfc": "^3.0.0",
    "@vue/eslint-config-prettier": "^6.0.0",
    "babel-eslint": "^10.1.0",
    "eslint": "^6.7.2",
    "eslint-plugin-prettier": "^3.3.1",
    "eslint-plugin-vue": "^7.0.0",
    "prettier": "^2.2.1",
    "vue-cli-plugin-vite": "~1.3.5"
  },
  "eslintConfig": {
    "root": true,
    "env": {
      "node": true
    },
    "extends": [
      "plugin:vue/vue3-essential",
      "eslint:recommended",
      "@vue/prettier"
    ],
    "parserOptions": {
      "parser": "babel-eslint"
    },
    "rules": {}
  },
  "browserslist": [
    "> 1%",
    "last 2 versions",
    "not dead"
  ]
}
```

```js
// ./src/main.js

import { createApp } from "vue";
import App from "./App.vue";
import router from "./router";
import "./index.css";

createApp(App).use(router).mount("#app");
```

**Comment by simonswiss** [comment]
Hey, are you able to share a repo with the code?

Your `package.json` file shows you're using the `vue-cli` and then vite through `vue-cli-plugin-vite`, so that's a different setup than the instruction guide you're linking to.

Happy to have a look, but it's pretty hard to tell by looking at just a `package.json` file.

PS: I edited your question to format the code snippets properly so it's easier to read.

**Reply by NTKrieger** [reply]
https://github.com/OmegaNix/Metamind

Thanks so much!  I've worked with all these technologies extensively but this is my first time trying to start a new project and it's been so frustrating.

**Reply by simonswiss** [reply]
Hey!

So... I cloned your repo, ran `yarn`, yarn vite`, added a couple of classes on the `/about` view, and it worked straight out of the box 😅

![image](https://user-images.githubusercontent.com/485747/127962815-5533f7ac-98c3-4ab3-995c-5403b28352a5.png)

As I commented in the PR, do you _really_ need to use the `vue-cli-service` here? If not, you're better off just using vite, as per the installation guide you linked to in the OP.

https://github.com/OmegaNix/Metamind/pull/1
